target/linux/ar71xx/patches-4.4/701-MIPS-ath79-add-routerboard-detection.patch

```diff
@@ -13,6 +13,7 @@
 +	    strstr(arcs_cmdline, "board=493") ||
 +	    strstr(arcs_cmdline, "board=951G") ||
 +	    strstr(arcs_cmdline, "board=2011L") ||
++	    strstr(arcs_cmdline, "board=2011r5") ||
```

Comment:
I was thinking the same, but since i have no idea what this actually does, i didnt want to enable it on boards that i dont have for testing.   Would you say it's ok to enable this more globally?